roles/openshift_logging_elasticsearch/templates/es.j2

```diff
@@ -90,6 +90,9 @@ spec:
               name: "RECOVER_AFTER_TIME"
               value: "{{openshift_logging_elasticsearch_recover_after_time}}"
             -
+              name: "READINESS_PROBE_TIMEOUT"
+              value: "30"
```

Comment:
@portante @t0ffel Any thoughts if this timeout is long enough?

Reply:
I've seen ES take much longer than 30 seconds to become available.

What happens after this 30 second timeout is hit?  Will Elasticsearch continue to recover indices, etc. whatever it is doing in the background?  Let's say it takes 1 hour to become ready - what does the user need to do to make the Elasticsearch service available in the cluster at that point?  Redeploy?

Reply:
Doesn't https://github.com/openshift/openshift-ansible/pull/5209/files#diff-0b7373a8acdac264abe27e0f9ded4f1aR117 say that readiness probe will timeout after 30 seconds, so if this is 30 seconds, then total time of readiness probe run may exceed 30 seconds and it'll time out?

Should that even be a parameter? imho hardcoding it is fine. If the 1st readiness probe fails, the next one or the one after next will succeed.

Reply:
these timeouts were picked from the exact original values that were previously in master - https://github.com/openshift/openshift-ansible/pull/4641/files